tikv-configuration-file.md

```diff
@@ -490,8 +490,9 @@ Configuration items related to Raftstore.
 
 + The soft limit on the size of a single message packet
 + Default value: `"1MB"`
-+ Minimum value: `0`
-+ Unit: MB
++ Minimum value: greater than `0`
```

Comment:
```suggestion
+ Minimum value: Greater than `0`
```

Reply:
> 

How about using the lower cases as other lines (for example 501, 538)?